cmd/dm-master/main.go

```diff
@@ -59,9 +60,8 @@ func main() {
 	utils.LogHTTPProxies(true)
 
 	// 3. print process version information
-	utils.PrintInfo("dm-master", func() {
-		log.L().Info("", zap.Stringer("dm-master config", cfg))
-	})
+	version.LogVersionInfo("DM Master")
```

Comment:
prefer we keep the old name for compatibility

Reply:
Done.